doc/lsp.txt

```diff
@@ -393,11 +393,17 @@ ignoreMissingServer	|Boolean| option.  Do not print a missing language
 						*lsp-opt-keepFocusInReferences*
 keepFocusInReferences	|Boolean| option.  Focus on the location list window
 			after LspShowReferences.
+			By default this is set to false.
+						*lsp-opt-noAdditionalTextEdits*
+noAdditionalTextEdits	|Boolean| option.  If not apply LSP server-initiated
+			actions after completion, specially if some others
+			plugins had registered to do that.
```

Comment:
Can you use the text from the above comment and update the help text?

Reply:
ok.